src/org/simpleyaml/configuration/file/YamlConfiguration.java

```diff
@@ -29,19 +25,10 @@
 
     protected static final String BLANK_CONFIG = "{}\n";
 
-    private final DumperOptions yamlOptions = new DumperOptions();
-    private final Representer yamlRepresenter = new YamlRepresenter();
-    private final Yaml yaml = new Yaml(new YamlConstructor(), yamlRepresenter, yamlOptions);
-
 	@Override
     public String saveToString() {
-        yamlOptions.setIndent(options().indent());
```

Comment:
There is some way to customize indentation with eo-yaml?

Currently with SnakeYAML and Simple-YAML you can do it with:

```java
yamlFile.options().indent(4);
```

The configured indentation is then set in this line to DumperOptions.